lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -177,7 +177,10 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
     |> Enum.map(fn %{informed_entities: ies} = alert ->
       distance =
         ies
-        |> Enum.filter(&String.starts_with?(&1.stop, "place-"))
+        |> Enum.filter(fn
+          %{stop: nil} -> true
```

Comment:
It's necessary so that way we can assign a distance. I suppose I could do an if/else like `if Enum.any?(ies, &is_nil(&1.stop)), do: 0` and prevent this logic from being executed. What do you think?

Reply:
Ohhhhh ok, I think I wasn't fully understanding this on the first read.
Do we assign a distance of 0 to IEs that don't have a stop, because we assume that they inform the whole route? (and thus the home stop, so 0 stops away?)

Reply:
^ (Answered in slack: Yes)